ShopApp/ShopApp/HomeViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  HomeViewController.swift
+//  ShopApp
+//
+//  Created by Andreea Sburlea on 27.05.2022.
+//
+
+import UIKit
+import FirebaseAuth
+
+class HomeViewController: UIViewController {
+
+    @IBAction private func logout() {
+        // Sign out from Google account
+        let firebaseAuth = Auth.auth()
+        do {
+            try firebaseAuth.signOut()
+        } catch let signOutError as NSError {
+            print("Error signing out: %@", signOutError)
+        }
+
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let firstView = storyboard.instantiateViewController(identifier: "ViewController")
```

Comment:
the login ViewController should be renamed to "LoginViewController" instead of "ViewController"and this "firstView" variable as well. "loginViewController"